<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="window">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>671</width>
      <height>441</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="window_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border: 0.5px dashed;
	border-radius: 10px;
	border-top-color: black; 
	border-left-color: black;
	border-right-color: black;
	border-bottom-color: black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_grip_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 13pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Profile Overview</string>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	border: 0px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
}</string>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	border: 10px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Total Time:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="total_time">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>hr</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>118</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 8px;
border: 1px solid;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	border: 0px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
}</string>
      </property>
      <property name="text">
       <string>Music Preferences:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="classic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
background-color: rgb(252, 252, 252);
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Classical Music</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="electronic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
background-color: rgb(252, 252, 252);
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;</string>
      </property>
      <property name="text">
       <string>Electronic Music</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="lofi">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
background-color: rgb(252, 252, 252);</string>
      </property>
      <property name="text">
       <string>Lo-Fi Music</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>118</width>
        <height>3</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 8px;
border: 1px solid;</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="am_bt">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: rgb(252, 252, 252);
	color: #edb4a1;
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
}
QPushButton::hover{
background-color: rgb(240, 240, 240);}
QPushButton::disabled {
	color: rgb(108, 108, 108);
	background-color: rgb(163, 163, 163);
}</string>
      </property>
      <property name="text">
       <string>☀️</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pm_bt">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	color: #edb4a1;
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
background-color: rgb(252, 252, 252);
}
QPushButton::hover{
background-color: rgb(240, 240, 240);}
QPushButton::disabled {
	color: rgb(108, 108, 108);
	background-color: rgb(163, 163, 163);
}</string>
      </property>
      <property name="text">
       <string>🌚</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="time_input">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="window_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border: 0.5px dashed;
	border-radius: 10px;
	border-top-color: black; 
	border-left-color: black;
	border-right-color: black;
	border-bottom-color: black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_grip_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="min">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>16</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	background-color: rgb(252, 252, 252);
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	border: 0px;
	color: #edb4a1;
	font: 25 13pt &quot;Cascadia Code Light&quot;;
}</string>
      </property>
      <property name="text">
       <string>Timer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_bt">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>16</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	color: #edb4a1;
background-color: rgb(252, 252, 252);
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
}
QPushButton::hover {
	color: rgb(203, 154, 138);
	background-color: rgb(240, 240, 240);
}
QPushButton::disabled {
	color: rgb(108, 108, 108);
	background-color: rgb(163, 163, 163);
}</string>
      </property>
      <property name="text">
       <string>▷</string>
      </property>
     </widget>
     <widget class="QPushButton" name="music_bt">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>16</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	color: rgb(72, 112, 86);
background-color: rgb(252, 252, 252);
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
}
QPushButton::hover {
	color: rgb(203, 154, 138);
	background-color: rgb(240, 240, 240);
}
QPushButton::disabled {
	color: rgb(108, 108, 108);
	background-color: rgb(163, 163, 163);
}</string>
      </property>
      <property name="text">
       <string>🎼</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="project_name">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>16</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	background-color: rgb(252, 252, 252);
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>Project Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_12">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>9</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sec">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>16</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	background-color: rgb(252, 252, 252);
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="window_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>421</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	border: 0.5px dashed;
	border-radius: 10px;
	border-top-color: black; 
	border-left-color: black;
	border-right-color: black;
	border-bottom-color: black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_grip_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>270</y>
        <width>19</width>
        <height>19</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="enter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	color: #edb4a1;
	background-color: rgb(252, 252, 252);
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
}
QPushButton::hover {
	color: rgb(203, 154, 138);
	background-color:  rgb(240, 240, 240);
}
QPushButton::disabled {
	color: rgb(108, 108, 108);
	background-color: rgb(163, 163, 163);
}</string>
      </property>
      <property name="text">
       <string>⇒</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	color: rgb(80, 125, 97);
	background-color: rgb(252, 252, 252);
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
}
QPushButton::hover {
	background-color: rgb(240, 240, 240);
}</string>
      </property>
      <property name="text">
       <string>🧹</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="task_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>-10</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 13pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>Task List</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="task_num">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	border-radius: 3px;
	color: #edb4a1;
	padding: 2px;
	padding-bottom: 5px;	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="DNALabel_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	border-radius: 3px;
	color: #edb4a1;
	font: 25 11pt &quot;Cascadia Code Light&quot;;
border: 0px;
}</string>
      </property>
      <property name="text">
       <string>#</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_bt">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border-radius: 5px;
	color: rgb(80, 125, 97);
	background-color: rgb(255, 0, 0);
	font: 11pt &quot;Cascadia Code&quot;;
	padding-bottom: 2px;
}

QPushButton:hover{
	background-color: rgb(222, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>🗑️</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="all_tasks">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px solid;
border-color: none;
border-radius: 6px;
	color: #edb4a1;
	
	font: 63 8pt &quot;Cascadia Code SemiBold&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Task 1:</string>
      </property>
     </widget>
     <zorder>frame_grip_5</zorder>
     <zorder>enter</zorder>
     <zorder>clear</zorder>
     <zorder>task_name</zorder>
     <zorder>DNALabel_6</zorder>
     <zorder>DNALabel_7</zorder>
     <zorder>task_num</zorder>
     <zorder>delete_bt</zorder>
     <zorder>all_tasks</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
